hibernate-core/src/test/java/org/hibernate/orm/test/jpa/criteria/QueryBuilderTest.java

```diff
@@ -260,8 +262,13 @@ public void testFunctionDialectFunctions() {
 		em.getTransaction().begin();
 		CriteriaBuilder cb = em.getCriteriaBuilder();
 		CriteriaQuery<Long> criteria = cb.createQuery( Long.class );
-		criteria.select( cb.count( cb.literal( 1 ) ) );
 		Root<Customer> root = criteria.from( Customer.class );
+		if (getDialect() instanceof InformixDialect && getDialect().getVersion().isBefore( 11,70 ) ) {
+			criteria.select( cb.count( root ) );
+		}
+		else {
+			criteria.select( cb.count( cb.literal( 1 ) ) );
+		}
```

Comment:
Sorry, but this change is a no-go. If you want to support this test, you will have to take care of how the `count` function is rendered. You can extend `org.hibernate.dialect.function.CountFunction` and register that custom subtype in `Dialect#initializeFunctionRegistry`.

Reply:
OK